apps/_pomodoro_app/realtime/pomodoroSocketIOServer.js

```diff
@@ -0,0 +1,104 @@
+const Timer = require('./application/timer')
+
+
+/**
+ * Socket auth middleware
+ * https://socket.io/docs/migrating-from-0-9/
+ * @param socket
+ * @param next
+ */
+function auth(socket, next) {
+    const handshakeData = socket.request;
+    console.log('auth-d', handshakeData._query['timer'])
```

Comment:
there is no validations and no checks. Is it OK?

Reply:
Yep, this is just a stab, that might be replaced with acutal authentication code on production, deicded to leave it there